Add LlamaIndex.rst

bug-orz opened the pull request:
Adding Qwen 1.5 examples on LlamaIndex

JianxinMa commented:
Thanks for your contributions. I have just resolved the merge conflict. 

@JustinLin610 Regarding the documentation about llamaindex, I noticed there's a mention of "We recommend using the bge-base-en-v1.5 model." Do you think it's appropriate to include references to other teams' models in our official documentation?

By the way, what is the purpose of the Colab file? It seems that it won't be displayed in the rendered documentation.

bug-orz commented:
> Thanks for your contributions. I have just resolved the merge conflict.
> 
> @JustinLin610 Regarding the documentation about llamaindex, I noticed there's a mention of "We recommend using the bge-base-en-v1.5 model." Do you think it's appropriate to include references to other teams' models in our official documentation?
> 
> By the way, what is the purpose of the Colab file? It seems that it won't be displayed in the rendered documentation.

Thank you for bringing this to our attention. We appreciate your feedback.

I'm glad to inform you that we have already completed these updates.  We added an online Colab link, modified phrases such as "we recommend," and included a Chinese translation version of our .rst file.

Please feel free to check them out and let us know if you have any further suggestions or feedback.

JustinLin610 commented:
Thx for the contrib! LGTM. Get it merged!
